configuration/index.md

```diff
@@ -24,26 +24,30 @@ In order to represent all of these, openHAB defines the following base component
 - [Rules](rules-dsl.html) - Automation logic, the "smart" in your Smart Home!
 - [JSR223 Scripting](jsr223.html) - Define rules and other runtime objects using [Javascript](http://openjdk.java.net/projects/nashorn/), [Jython](http://www.jython.org) or [Groovy](http://www.groovy-lang.org/)
 
-The individual articles hold all details needed to understand the concepts behind these building bricks for your Smart Home.
+The individual articles have all the details needed to understand the concepts behind these building bricks for your Smart Home.
 For more details on the base concept behind openHAB, please visit the [Concepts Overview page](/docs/concepts/index.html).
 
 ## Versatility
 
-openHAB provides different user interfaces to modify settings, manage your things and items and to build rules and sitemaps.
-HABmin provides a set of features to now overlap with Paper UI (which came later).
-Every interface still has its own strong points.
-The following table shows what the user interfaces can be used for.
-For a recommendation to new users which one(s) to use, see the section below. In the table, **"R" means recommended.**
+openHAB 3 now provides a _single_ graphical user interface to modify settings, manage your components and to manage rules and graphical user interfaces for users.
```

Comment:
I would love to remove all version references on long term but i am fine with keeping them during release of a new main version.

Reply:
yes that was my thinking, too. We should prominently and explicitly mention things that change from OH2 to 3 so longterm users that migrate can just 'scan' the official docs for things they need to know about and we don't need another document for them. These parts can be removed in the long run.
What about using a "::: tip for OH2 users" tag ?

Reply:
That would be feasible, since we could bulk remove them at some point when they are done in a similar way. 👍